tests/test_settings.py

```diff
@@ -61,3 +64,21 @@ def settings_tidy_concurrent(tidy: bool):
         tidy_array = list(p.map(settings_tidy_concurrent, random_booleans))
 
     assert random_booleans == tidy_array
+
+
+@pytest.mark.cflake
+def test_settings_cache_disable(caplog):
+    """Check Settings object with default cache_disable in env"""
+    caplog.set_level(logging.INFO)
+
+    import os
+
+    os.environ["WD_CACHE_DISABLE"] = "True"
+
+    from wetterdienst import Settings
+
+    Settings.reset()
+
+    assert Settings.cache_disable
```

Comment:
I did not have a closer look yet, but this looks like it is mixing up access via class methods vs. instance methods.

Reply:
Apparently the instance is directly instantiated at https://github.com/earthobservations/wetterdienst/blob/99b34e9db91bfe3c9707304b8e854f311082cd9f/wetterdienst/settings.py#L81
so that the import uses the same instance on every level. Do you think this is okay or is it bad design?